robots/configBasedRobot.py

```diff
@@ -29,15 +34,57 @@ def __init__(self, period: float = 0.02) -> None:
 
         self.navx = navx._navx.AHRS.create_spi()
 
-    def getAutonomousCommand(self):
-        return(Autonomous(self.driveTrain, self.navx))
+        # We should only need to do this once
+        wpilib.SmartDashboard.putString("Robot Version", self.getDeployInfo("git-hash"))
```

Comment:
While we are here we should also add config file name

Reply:
On it!